src/Tasks/Microsoft.NET.Build.Tasks/DependencyContextBuilder.cs

```diff
@@ -32,7 +32,7 @@ public DependencyContextBuilder(SingleProjectInfo mainProjectInfo, ProjectContex
             _projectContext = projectContext;
 
             // This resolver is only used for building file names, so that base path is not required.
-            _versionFolderPathResolver = new VersionFolderPathResolver(path: null);
+            _versionFolderPathResolver = new VersionFolderPathResolver(null);
```

Comment:
I left the parameter name out so that it will still compile locally if you have the old version of the package in your packages folder.